docs/pages/web/pulsar.js

```diff
@@ -44,6 +53,128 @@ export default function DocsPage({ generatedDocGen }: {| generatedDocGen: DocGen
         </MainSection.Subsection>
       </MainSection>
 
+      <MainSection name="Best Practices">
+        <MainSection.Subsection columns={2}>
+          <MainSection.Card
+            cardSize="md"
+            type="do"
+            description={`
+            Show Pulsar with an educational [Popover](/web/popover) that contains an affordance to dismiss both Popover and Pulsar.
+            `}
+            sandpackExample={
+              <SandpackExample
+                code={doPopover}
+                name="Do - Popover"
+                hideEditor
+                previewHeight={bestPracticesHeight}
```

Comment:
Yo dont need 600 for these examples. 

This example is not showing. Popover must be "close" and then "open" to be positioned. 

Set state to false.

Reply:
AHHHH thank you so much! I had no idea. I'll add a note to Popover's docs as well about this for "static" popovers like this. Not intuitive at all. Thank you!!